development.md

```diff
@@ -0,0 +1,20 @@
+# Development
+
+You'll need [Go installed][go] to be able to contribute to `catalog-importer`.
+
+You can run all of the tests using `go test`:
+
+```
+go test ./...
+```
+
+To build the binary, run `make`: this will place a built version of the tool in
+the `bin` directory. If you have a local instance of incident.io running, then
```

Comment:
We might want to rephrase this slightly to make it clear only incident staff should be configuring this?